File: code/95098.pas
program Bai098;
var n , k: integer;
begin
  readln(n);
  if n mod 3 <> 0 then writeln('Khong the') else
  begin
  k := n div 3;
  write(k - 1, '  ', k, '  ', k + 1);
  end;
end.

File: code/95098.cpp
#include <iostream>
using namespace std;

int main() 
{
    int n,k;
    cin>>n;
    if (n%3!=0)
        cout<<"KHONG THE";
    else k=n/3;
    cout<<k-1<<" "<<k<<" "<<k+1;
    return 0;
}